src/data/finset/lattice.lean

```diff
@@ -249,6 +249,14 @@ le_antisymm
 lemma sup_id_eq_Sup [complete_lattice α] (s : finset α) : s.sup id = Sup s :=
 by simp [Sup_eq_supr, sup_eq_supr]
 
+lemma sup_id_set_eq_sUnion (s : finset (set α)) :
+  s.sup id = ⋃₀(↑s) :=
+sup_id_eq_Sup _
+
+@[simp] lemma sup_set_eq_bUnion (s : finset α) (f : α → set β) :
+  s.sup f = ⋃ x ∈ s, f x :=
+sup_eq_supr _ _
```

Comment:
For consistency with below:
```suggestion
@[simp] lemma sup_id_set_eq_sUnion (s : finset (set α)) : s.sup id = ⋃₀(↑s) :=
sup_id_eq_Sup _

@[simp] lemma sup_set_eq_bUnion (s : finset α) (f : α → set β) : s.sup f = ⋃ x ∈ s, f x :=
sup_eq_supr _ _
```
(`simp` and line-unwrapping)

Reply:
OK.

Reply:
This comment still applies

Reply:
It seems it is OK now.